unidist/core/backends/mpi/core/monitor.py

```diff
@@ -9,6 +9,7 @@
 
 # MPI stuff
 comm, rank, world_size = communication.get_mpi_state()
+topology = communication.get_topology()
```

Comment:
Why do we need this here?

Reply:
It's need because `communication.get_topology()` has `allgather` call, which should be called from each rank of MPI cluster.

Reply:
ok